src/hooks/useColorMode.ts

```diff
@@ -0,0 +1,20 @@
+import { ColorScheme } from '@mantine/core';
+import { useColorScheme, useHotkeys, useLocalStorage } from '@mantine/hooks';
+
+const useColorMode = () => {
+  const preferredColorScheme = useColorScheme('light', { getInitialValueInEffect:true});
+  const [colorScheme, setColorScheme] = useLocalStorage<ColorScheme | 'preferredColorScheme'>({
+    key: 'tfiltr-color-scheme',
+    defaultValue: 'preferredColorScheme',
+    getInitialValueInEffect: true,
+  });
+
+  const toggleColorScheme = (value?: ColorScheme|'preferredColorScheme') => setColorScheme(value || (colorScheme === 'dark' ? 'light' : 'dark'));
+
+  useHotkeys([['mod+J', () => toggleColorScheme()]]);
+  useHotkeys([['mod+shift+J', () => toggleColorScheme('preferredColorScheme')]]);
+
+  return { colorMode: colorScheme !== 'preferredColorScheme'? colorScheme: preferredColorScheme, toggleColorMode: toggleColorScheme };
+};
```

Comment:
this is the new color mode, I think this works well. The hotkey can be changes if needed tho

Reply:
this way it stays in sync with the device

Reply:
Looks good